config/config.exs

```diff
@@ -18,6 +18,16 @@ import Config
 #
 
 config :main_app,
-  subscriber: PgSubscriber.Handler,
-  replicator: PgSubscriber.Repl,
-  publisher: FilePublisher
+  replicator: PgSubscriber.Subscriber,
```

Comment:
I feel like you should have picked `@subscriber` instead of `@replicator`. The app name is `pg_**subscriber**` and the module's name is `Pg**Subscriber**.**Subscriber**`.